ParisBNF/abb/BNFabb89.xml

```diff
@@ -0,0 +1,325 @@
+                                                ref="LOC2866EndaAb">Endā abbā Garimā</placeName>
+                                            (see, for example, <title>Lettere annue di Ethiopia del
+                                                1624, 1625 e 1626</title>, Rome, 1628, first
+                                            letter). Obviously, the <ref type="mss"
+                                                corresp="PetermannIINachtrag28"> ms. Peterm. II
+                                                Nachtr. 28 </ref> in Berlin, which contains this
+                                            account, cannot be from the 16th century, as <persName
+                                                ref="PRS3562Dillmann">Aug. Dillmann</persName>
+                                            believed. However, it already includes the curse: "May
+                                            your offering be like the offering of Rome" (see Conti
+                                            Rossini, v. 524-525). It should be noted that the
+                                                <foreign xml:lang="gez">gadl</foreign> in our
+                                            manuscript begins with the well-known formula from the
+                                            time of King <persName ref="PRS10626ZaraY">Zarʾa
+                                                Yāʿqob</persName>: <foreign xml:lang="gez">በስመ፡ አብ፡,
+                                                etc. በቅድስት፡ ሥላሴ፡ እንዘ፡ እት<gap reason="illegible"/>፡
+                                                ወእትመኀፀን፡ እክሕደከ፡ ሰይጣን፡ በቅድመ፡ ይእቲ፡ እምየ፡</foreign>
+                                            which would confirm the age of this work.</quote>
+                                    </cit> (<cit>
+                                        <quote>On y trouve le récit du serpent roi de la terre
+                                            d'Agʿāzi, <foreign xml:lang="gez">ብሔረ፡ አግዓዚ፡, mais les
+                                                concessions des fiefs semblent manquer. Le récit
+                                                final sur les malheurs qui devront frapper
+                                                  <placeName ref="LOC2254DabraD">le couvent de
+                                                  Dāmo</placeName> manque aussi; on voit qu'il
+                                                s'agit d'une addition plus récent, qui vise sans
+                                                doute les ravages des catholiques à <placeName
+                                                  ref="LOC2866EndaAb">Endā abbā Garimā </placeName>
+                                                (voir p.ex. <title>Lettre annue di Ethiopia del
+                                                  1624, 1625 e 1626, Rome, 1628, première
+                                                  lettre</title>). Évidemment, le <ref type="mss"
```

Comment:
I do not understand where this reference is closed, `</ref>` should come right after the shelfmark